ga4gh/datamodel/variants.py

```diff
@@ -269,6 +269,7 @@ class HtslibVariantSet(datamodel.PysamDatamodelMixin, AbstractVariantSet):
     """
     def __init__(self, id_, dataDir):
         super(HtslibVariantSet, self).__init__(id_)
+        self._dataDir = dataDir
```

Comment:
I agree that we shouldn't have an init in mixins (that way madness lies). Let's leave it as it is and see if we can tidy the concepts up a bit in our next refactor.